eodatasets/prepare/s2_l1c_aws_pds_prepare.py

```diff
@@ -154,27 +156,35 @@ def get_datastrip_info(path):
             uuid.NAMESPACE_URL,
             datastrip_metadata + "#" + product_info['id']
         )
-    return datastrip_metadata, persisted_uuid
+    return (datastrip_metadata, persisted_uuid)
 
 
 def get_tile_info(path):
+    # type Path -> str
     """get_tile_info: returns information parsed from the tileInfo.json
 
     :param path: path to the root of the tile collection
     :returns: prefix to tile collection
     """
     with (path / 'tileInfo.json').open() as fd:
         tile_info = json.load(fd)
-        tile_path = tile_info.get('path')
-    return tile_path
+        return tile_info.get('path')
 
 
-def prepare_dataset(path):
+def prepare_dataset(path, datastrip_path=None):
+    # type (Path, Optional(Path)) -> Dict
```

Comment:
Returns a list of documents (PyCharm shows a warning about this)

```suggestion
    # type: (Path, Optional[Path]) -> List[Dict]
```


(And ideally ```from typing import Dict, Tuple, List, Optional``` at the top of the file for all of these docs)